sdks/python/apache_beam/utils/counters.py

```diff
@@ -28,18 +28,45 @@
 
 from apache_beam.transforms import cy_combiners
 
-# Information identifying the IO being measured by a counter.
```

Comment:
<!--new_thread; commit:edb8389d02012005d37366b9443052d8238fdec9; resolved:0-->
Can you keep this line?  The added lines clarify the intent, but this line is still helpful as an overall description of `IOTargetName`.